### PR TITLE
[13.0][IMP] web_favicon functionality merged in base

### PR DESCRIPTION
**Pull request by Tardo**
web_favicon functionality merged in base, updated base migration script to rename fields.

cc @tecnativa TT19825

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by MiquelRForgeFlow — APPROVED** [review]

File: odoo/addons/openupgrade_records/lib/apriori.py [hunk 1]
```diff
@@ -36,6 +36,7 @@
     'sale_timesheet_existing_project': 'sale_timesheet',
     # OCA/web
     'web_widget_color': 'web',
+    'web_favicon': 'base',
```
Comment (pedrobaeza) [inline odoo/addons/openupgrade_records/lib/apriori.py:38]: Please sort in alphabetical order for each repo (put this one up)

**Review by chienandalu — APPROVED** [review]
Onces requested changes are done :+1:

**Comment by Tardo** [comment]
Done!

**Pull request merged**